generatebundlefile/main.go

```diff
@@ -91,9 +91,13 @@ func cmdPromote(opts *Options) error {
 
 	promoteCharts := make(map[string][]string)
 
```

Comment:
this doesn't make much sense

Reply:
What doesn't make sense the `promoteCharts`?

Reply:
It was added for scenario's where we are promoting multiple charts for a single pipeline like cluster-autoscaler, metrics server, emissary, metallb and we are using an input file to specify which ones to promote.

Reply:
well, if you pass in a tag, it gets set to latest. This is either wrong or the tag should be a boolean type

Reply:
OOoooooooo, yea I did the logic backwards let me fix that.

Reply:
Gonna open a new PR with the fix, and the script changes actually.

Reply:
Fixed